bazel_tools/haskell-rts-docs.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/haskell/assets/ghc_9_0_2_win.patch b/haskell/assets/ghc_9_0_2_win.patch
```

Comment:
i heard you like patch files so here's a patch file for a patch file

Reply:
it’s patches all the way down